### PR TITLE
archivy: init at 0.8.5

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
@GrahamcOfBorg build archivy

**Comment by rmcgibbo** [comment]
I built this with `nix-review pr 103815`, and was able to open executable and start the application in the browser. LGTM.

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 103815` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>archivy</li>
    <li>python27Packages.python-frontmatter</li>
    <li>python37Packages.python-frontmatter</li>
    <li>python38Packages.python-frontmatter</li>
  </ul>
</details>

File: pkgs/development/python-modules/python-frontmatter/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    inherit pname version;
+    sha256 = "09n44qmzbzpgddgzmissaawa671k1w29j8mqv1l0r7wfqc7ykhm9";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    pyyaml
+    six
+  ];
+
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/python-modules/python-frontmatter/default.nix:17]: There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 103815` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>archivy</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python27Packages.python-frontmatter (python37Packages.python-frontmatter ,python38Packages.python-frontmatter)</li>
  </ul>
</details>

**Comment by siraben** [comment]
```
builder for '/nix/store/kp5qmxrq5rvpiqy42830lbmzbshfgz5q-python3.8-isort-4.3.21.drv' failed with exit code 1; last 10 log lines:
  >           assert str(tmpdir.join("file1.py")) in out
  E           AssertionError: assert '/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0/file1.py' in 'Fixing /private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0/file2.py\n'
  E            +  where '/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0/file1.py' = str(local('/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0/file1.py'))
  E            +    where local('/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0/file1.py') = <bound method LocalPath.join of local('/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0')>('file1.py')
  E            +      where <bound method LocalPath.join of local('/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0')> = local('/private/tmp/pytest-of-siraben/pytest-2/test_command_line_True_0').join
  
  /private/tmp/nix-build-python3.8-isort-4.3.21.drv-0/isort-4.3.21/test_isort.py:2759: AssertionError
  =========================== short test summary info ============================
  FAILED test_isort.py::test_command_line[True] - AssertionError: assert '/priv...
  ================= 1 failed, 153 passed, 2 deselected in 5.57s ==================
cannot build derivation '/nix/store/0nsqjf5b1hk26syc33k0wr9zm0ygabqq-python3.8-poetry-core-1.0.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/1jahrni7z0x2zr4pii0ajyjkvhl637hw-python3.8-validators-0.17.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/2acjmf3z78wxw6yw71g5wp8jgsdf915m-python3.8-poetry-1.1.4.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/kjf1vfwmymg79q6mnxg0c50pn054b62w-python3.8-tinydb-4.1.1.drv': 1 dependencies couldn't be built
builder for '/nix/store/bx8w80rv5gma0pkkvjr3afic7zjxkaaq-python3.8-pypandoc-1.5.drv' failed with exit code 1; last 10 log lines:
  ?       -
  + # üäöîôû? ++
  - ======
  
  ----------------------------------------------------------------------
  Ran 24 tests in 8.572s
  
  FAILED (failures=1, skipped=1)
  Test failed: <unittest.runner.TextTestResult run=24 errors=0 failures=1>
  error: Test failed: <unittest.runner.TextTestResult run=24 errors=0 failures=1>
cannot build derivation '/nix/store/fw65fhfd2kj608zswfhyw1nk8ai1jb5x-archivy-0.8.4.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/5kf5i03yg4bfqf5ak3diprkbrcq6w6c9-env.drv': 1 dependencies couldn't be built
```

**Comment by siraben** [comment]
Looks like pypandoc tests failing would be fixed by #104625

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 103815` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>archivy</li>
    <li>python27Packages.python-frontmatter (python37Packages.python-frontmatter ,python38Packages.python-frontmatter)</li>
  </ul>
</details>

**Pull request merged**